packages/hdwallet-phantom/src/types.ts

```diff
@@ -0,0 +1,7 @@
+import { providers } from "ethers";
+
+export type PhantomEvmProvider = providers.ExternalProvider & {
+  _metamask: {
+    isUnlocked: () => boolean;
+  };
```

Comment:
Phantom injects this on top of regular provider things, we need this to detect unlocked state. Seemingly called `_metamask` since that's conventionally a MM property